channeldb/channel.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/boltdb/bolt"
+	"github.com/lightningnetwork/lnd/lnwire"
```

Comment:
In light of my comment above, the entire diff within this file should be reverted. The database serialization code shouldn't depend on the wire protocol framing.